ads/_a4a-config.js

```diff
@@ -57,3 +57,12 @@ export const a4aRegistry = {
 if (getMode().localDev || getMode().test) {
   a4aRegistry['fake'] = fakeIsA4AEnabled;
 }
+
+/**
+ * An object mapping signing server names to their corresponding URLs.
+ * @type {!Object<string, string>}
+ */
+export const signingServerURLs = {
```

Comment:
Yes, I believe the plan is to allow ad networks to add their own verification URL if they so wish.